EZSwiftExtensionsTests/BoolTests.swift

```diff
@@ -22,5 +22,13 @@ class BoolTests: XCTestCase {
         value.toggle()
         XCTAssertNotEqual(value, false)
     }
+    
+    func testToggled() {
+        let value = false
```

Comment:
if the other variable is trueVal, this one should be falseVal to keep consistency